.github/workflows/test-runner-on-perf.yml

```diff
@@ -68,7 +69,7 @@ jobs:
           echo "Building Data Repo client library"
           ../gradlew clean assemble
           cd ${GITHUB_WORKSPACE}/${workingDir}/datarepo-clienttests
-          export ORG_GRADLE_PROJECT_datarepoclientjar=$(find .. -type f -name "datarepo-client-*.jar")
+          export ORG_GRADLE_PROJECT_datarepoclientjar=$(find .. -type f -name "datarepo-client*.jar")
```

Comment:
This is the only change that will actually be merged right?

Reply:
Correct - This and the GitHub action version bump.

Reply:
Got it, thanks.